web/resources/api/api.oas3.yaml

```diff
@@ -3193,7 +3193,8 @@ components:
       required: [vote]
       properties:
         vote:
-          allOf: ['#/components/schemas/CastVote']
```

Comment:
If you're gonna do formatting changes, make that a separate PR as that's endless bikeshedding. Most of the formatting choices I made were to preserve my sanity as I basically wrote this entire thing myself lol.

Reply:
Wait I'm a moron, disregard

Reply:
Gotcha, i only noticed this due to swagger complaining, so i put it all together into one pr :)